x/auth/legacy/v0_38/types.go

```diff
@@ -46,15 +46,15 @@ type (
 
 	BaseAccount struct {
 		Address       sdk.AccAddress `json:"address" yaml:"address"`
-		Coins         sdk.Coins      `json:"coins" yaml:"coins"`
+		Coins         sdk.Coins      `json:"coins,omitempty" yaml:"coins,omitempty"`
```

Comment:
I've decided to add `omitempty` here to avoid having to literally copy the entire file solely to remove a single field.

Reply:
Like this move!